app/controllers/course/statistics/answers_controller.rb

```diff
@@ -39,10 +39,10 @@ def question_answer_details
                                              { discussion_topic: { posts: :codaveri_feedback } } } },
                                     discussion_topic: { posts: :codaveri_feedback }).first
 
-    fetch_all_answers(submission_id, question_id, true)
+    fetch_all_answers(submission_id, question_id, answer_params[:limit].to_i)
   end
 
-  def all_answers
+  def all_attempts
```

Comment:
Similar comment, `all_answers` vs `all_attempts`

Reply:
Attempt aligns with the vocab used in frontend, since we have never exposed the concept of "answer" to users